analyzer/java/com/engflow/bazel/invocation/analyzer/bazelprofile/ThreadId.java

```diff
@@ -15,17 +15,18 @@
 package com.engflow.bazel.invocation.analyzer.bazelprofile;
 
 import java.util.Objects;
+import javax.annotation.Nullable;
 
 public class ThreadId {
   private final int processId;
-  private final int threadId;
+  private final Integer threadId;
 
-  public ThreadId(int processId, int threadId) {
+  public ThreadId(int processId, @Nullable Integer threadId) {
     this.processId = processId;
     this.threadId = threadId;
   }
 
-  public int getThreadId() {
+  public Integer getThreadId() {
```

Comment:
Should this be marked `@Nullable`?

Reply:
I am not planning to merge this PR (hence I converted it to a draft), given that the issue is hopefully resolved on the Bazel side (ensuring profiles do have `tid` for action count entries) - see https://github.com/bazelbuild/bazel/pull/20300.